src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/NcwmsStyle.java

```diff
@@ -0,0 +1,25 @@
+package au.org.emii.geoserver.wms;
+
+import java.util.ArrayList;
+
+public class NcwmsStyle {
+    private final ArrayList<String> styles;
```

Comment:
This class should use `List<String>` anywhere within it. Program to the `List<String>` interface, rather than using `ArrayList`, which is a specific implementation of `List`.